.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,19 +1,23 @@
-<!--
-Checklist for submitting a Pull Request
+<!--  Thanks for sending a pull request!  Here are some tips for you:
 
-Please remove this comment block before submitting.
+1. If this is your first time, read our contributor guidelines https://github.com/kubernetes/kubernetes/blob/master/CONTRIBUTING.md and developer guide https://github.com/kubernetes/kubernetes/blob/master/docs/devel/development.md
 
-1. Please read our [contributor guidelines](https://github.com/kubernetes/kubernetes/blob/master/CONTRIBUTING.md).
-2. See our [developer guide](https://github.com/kubernetes/kubernetes/blob/master/docs/devel/development.md).
-3. If you want this PR to automatically close an issue when it is merged,
-   add `fixes #<issue number>` or `fixes #<issue number>, fixes #<issue number>`
-   to close multiple issues (see: https://github.com/blog/1506-closing-issues-via-pull-requests).
-4. Follow the instructions for [labeling and writing a release note for this PR](https://github.com/kubernetes/kubernetes/blob/master/docs/devel/pull-requests.md#release-notes) in the block below.
+2. If you want *faster* PR reviews, read how: https://github.com/kubernetes/kubernetes/blob/master/docs/devel/faster_reviews.md
+
+3. Follow the instructions for writing a release note: https://github.com/kubernetes/kubernetes/blob/master/docs/devel/pull-requests.md#release-notes
 -->
 
+**What this PR does / why we need it**:
+
+**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #
+
+**Special notes for your reviewer**:
+
+**Release note**:
 ```release-note
```

Comment:
It would be nice if we could make the syntax clear enough to automatically set the release-note labels based on the text here.